Bottom: 3365e7f9e292693b892fe05f705c9491ed4be9a8
Top:    2f4acd57dbdf2f0bf58fc25a86580ebf5865ec1a
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 22:46:54 -0500

Refresh of rakefile

---

diff --git a/Rakefile b/Rakefile
new file mode 100644
index 0000000..8b91519
--- /dev/null
+++ b/Rakefile
@@ -0,0 +1,4 @@
+desc "Remove trailing whitespace"
+task :rm_whitespace do
+  `sed -i 's/[[:space:]]*$//' script.js syntax.php`
+end
